src/shogun/machine/gp/Inference.cpp

```diff
@@ -39,18 +39,11 @@
 #include <shogun/mathematics/Math.h>
 #include <shogun/lib/Lock.h>
 
-using namespace shogun;
+#ifdef HAVE_OPENMP
+#include <omp.h>
```

Comment:
@vigsterkr I didn't know that. Will make the chages. Thanks for pointing that out.